mappings/net/minecraft/datafixer/FixType_.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_2649 net/minecraft/datafixer/FixType_
```

Comment:
Not a fan of what you did here, but i will go through datafixers in a later pr so ig allow it